staging.kernelci.org

```diff
@@ -349,6 +349,7 @@ cmd_api_pipeline() {
     git prune
     docker-compose down
     echo "Starting API containers"
+    docker-compose build --no-cache
```

Comment:
Please correct me if I'm wrong: currently API should use prebuilt images from `cmd_docker()` (when docker-compose is adjusted to use new "name:tag" scheme) and only [`ssh` is still built from docker-compose context](https://github.com/kernelci/kernelci-api/blob/50fca4be6a0c4755cb5dc239f1e454ff22c300fe/docker-compose.yaml#L49).

Does it need a rebuild on each API containers restart?

Reply:
Yes, it's true, that we don't need to rebuild services every time as long as they are using the same image. 
But it didn't work when I supplied a different image and tag. Without the `build` command, API services were using the previously built images with `kernelci/staging-api`. That's why I needed to rebuild the services.